test/suites/upgrade/upgrade-successful.robot

```diff
@@ -21,14 +22,15 @@ ${TARGET_REF}       ${EMPTY}
 *** Test Cases ***
 Upgrade
     [Documentation]    Performs an upgrade to given reference
-    ...    and verifies if it was successful
+    ...    and verifies if it was successful, with SELinux validation
 
     MicroShift 413 Should Not Have Upgrade Artifacts
     Wait For Healthy System
 
     ${future_backup}=    Get Future Backup Name For Current Boot
     Deploy Commit Not Expecting A Rollback    ${TARGET_REF}
     Backup Should Exist    ${future_backup}
+    Validate SELinux With Backup    ${future_backup}
```

Comment:
I wonder if we'd better change `Backup Should Exist` library function to call SELinux check?
This should cover all the cases where it's called.

@pmtk , WDYT?